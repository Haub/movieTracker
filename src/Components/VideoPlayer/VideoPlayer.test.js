import React from 'react';
import { VideoPlayer } from './index';
import { shallow } from 'enzyme';

describe('VideoPlayer', () => {
let wrapper;
let play;
let url;
let image;
let feature;
  
  beforeEach(() => {
    wrapper = shallow(<VideoPlayer play={play} url={url} image={image} feature={feature} />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})