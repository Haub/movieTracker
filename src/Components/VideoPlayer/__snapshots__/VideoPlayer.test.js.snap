// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer should match the snapshot 1`] = `
<div
  className="background"
>
  <img
    alt="movie-background"
    className="poster"
    src="https://image.tmdb.org/t/p/w500/undefined"
  />
</div>
`;
